nbconvert/exporters/tests/test_exporter.py

```diff
@@ -73,3 +75,30 @@ def test_get_export_names_disable(self):
         })
         export_names = get_export_names(config=config)
         self.assertEqual(export_names, ['notebook'])
+
+    def test_get_exporter_disable_config_exporters(self):
+        """
+        Does get_export_names behave correctly with respect to
+        NBCONVERT_DISABLE_CONFIG_EXPORTERS being set in the
+        environment?
+        """
+        config = Config({
+            'Exporter': {'enabled': False},
+            'NotebookExporter': {'enabled': True}
+        })
+        os.environ["NBCONVERT_DISABLE_CONFIG_EXPORTERS"] = "1"
```

Comment:
Fun fact, you can use `patch.dict` to temporarily set values in `os.environ`, there is an example in the docs:

https://docs.python.org/3.7/library/unittest.mock.html#unittest.mock.patch.dict